StreamChatClient_v3/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -21,4 +21,12 @@ extension Endpoint {
               requiresConnectionId: query.options.contains(oneOf: [.presence, .state, .watch]),
               body: query)
     }
+
+    static func muteChannel(cid: ChannelId, mute: Bool) -> Endpoint<EmptyResponse> {
+        .init(path: "moderation/\(mute ? "mute" : "unmute")/channel",
+              method: .post,
+              queryItems: nil,
+              requiresConnectionId: true,
+              body: ["channel_cid": cid])
+    }
```

Comment:
This needs to be tested. We don't have tests for the existing endpoints either, sorry. Can you please add them too? Just simple tests that would verify the logic we have here.

Reply:
@VojtaStavik done